### DISCUSSION TITLE
Sensus Gas Meter Config

### DISCUSSION THREAD

**Discussion by dadcoachengineer** [jomjol/AI-on-the-edge-device#2998] [General] [answered]
Hey there!

Any guidance appreciated on how to configure for this Sensus analog diaphragm meter. Top four dials seems to be linear but the bottom two are in 1/2 and 2 CF increments.

![Screenshot 2024-03-16 at 5 50 04 PM](https://github.com/jomjol/AI-on-the-edge-device/assets/6666082/feeaf054-6674-459a-b85a-7477e5d57154)
![Screenshot 2024-03-16 at 5 54 50 PM](https://github.com/jomjol/AI-on-the-edge-device/assets/6666082/f9065768-fe07-41a5-8f07-13b93fde8ed2)

**Comment by caco3** [comment, marked as answer]
There is currently no support for non-0-9-scales of the analog pointers. You will need to stick to the top 4 ones and ignore the 2 bottom ones.

**Reply by dadcoachengineer** [reply]
Sounds good. I just submitted a labeled dataset as white background with black dials did not seem to being in the training data.
